src/components/stack/index.stories.mdx

```diff
@@ -0,0 +1,129 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs';
+
+import Stack from '.';
+
+<Meta title="Components/Stack" component={Stack} />
+
+# Stack
+
+The Stack component is a wrapper around the HTML Flex element.
+It allows for a both a column or row layout.  In addition,
+the Stack component allows for alignment of the items in the layout
+axis as well over-riding the default gap (also also know by the term "gutter") size.
+
+## Usage
+
+```
+import { Stack } from '@wpmedia/arc-themes-components';
+
+
+<Stack>Component Children</Stack>
+```
+
+## Properties
+
+<Props of={Stack} />
+
+## Stories
+
+** Default **
+<Preview>
+	<Story name="Default">
+		<Stack>Stack Text</Stack>
+	</Story>
+</Preview>
+
+
+** Vertical **
+<Preview>
+	<Story name="Vertical">
+		<Stack>
+			<div style={{ height: '100px', width: '200px' }}>
+			    <img alt="image 1" src="https://picsum.photos/200/100" />
```

Comment:
Couple of things to note with using third party image services with Chromatic is that they can give false positives on the snapshots as they don't always render in time for Chromatic - we then have to apply waiting times for chromatic tests to ensure we catch the image rendering.
Also with Chromatic and images, if they image isn't always the same (some placeholder image services return different images for the same url) we will get a false positive

Reply:
Using background colors for the stories